.github/workflows/release.yml

```diff
@@ -1,26 +1,28 @@
 name: Publish to PyPI
-on: 
+on:
   release:
     types: [published]
+  workflow_dispatch:
+
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
v3 is out. Can this use actions/checkout@v3? https://github.com/actions/checkout/releases/tag/v3.3.0